### PR TITLE
Fix node rendering for cables with hidden name and/or no visible attributes

### LINKED ISSUE #69: [bug] fix issue when hiding both name and wirecount of a cable/bundle with no further attributes

### PR DESCRIPTION
**Pull request by formatc1702**
Closes #69.

Probably just an interim solution until work on #66 continues, and a more elegant fix arises.

### CODE REVIEW

**Comment by formatc1702** [comment]
Diff might be best observed using [this link](https://github.com/formatc1702/WireViz/pull/104/files?w=1).
[Source](https://github.blog/2011-10-21-github-secrets/) for the hint of using `?w=1` at the end of the URL to ignore whitespace/indent changes.

File: src/wireviz/Harness.py [hunk 1]
```diff
@@ -172,23 +172,25 @@ def create_graph(self) -> Graph:
...
-            for attrib in attributes:
-                html = f'{html}<td balign="left">{attrib}</td>'
-            html = f'{html}</tr>'  # attribute row
-            html = f'{html}</table></td></tr>'  # name+attributes table
+            html = '<table border="0" cellspacing="0" cellpadding="0">'  # main table
+
+            if cable.show_name or len(attributes) > 0:
+                html = f'{html}<tr><td><table border="0" cellspacing="0" cellpadding="3" cellborder="1">'  # name+attributes table
+                if cable.show_name:
+                    html = '{html}<tr><td colspan="{colspan}">{name}</td></tr>'.format(html = html, colspan = len(attributes) if len(attributes) > 0 else 1, name = cable.name)
```
Comment (kvid) [inline src/wireviz/Harness.py:180]: Line 180 might be replaced by:
`html = f'{html}<tr><td colspan="{max(len(attributes), 1)}">{cable.name}</td></tr>'`

Comment (formatc1702) [reply src/wireviz/Harness.py]: Nice, added.

**Comment by formatc1702** [comment]
Let me know if you find anything else, please confirm with an approving review if not :)
Same for #103 and #105, even if they're simple changes, it's good to know at least someone looked at it before merging.
Cheers.

**Comment by kvid** [comment]
It's easier to approve this by verifying the fix if #69 would include a YAML file that demonstrates the bug. See https://github.com/formatc1702/WireViz/issues/69#issuecomment-660943712.

**Pull request merged**